web/client/components/map/openlayers/Layer.jsx

```diff
@@ -259,13 +259,11 @@ export default class OpenlayersLayer extends React.Component {
                     imageStopStream$.next();
                 }
             });
-            this.layer.getSource().on('imageloaderror', (event) => {
-                this.imagestoload--;
-                imageLoadEndStream$.next({type: 'imageloaderror', event});
-                if (this.imagestoload === 0) {
-                    imageStopStream$.next();
-                }
-            });
+            this.imagestoload--;
+            imageLoadEndStream$.next({type: 'imageloaderror', event});
+            if (this.imagestoload === 0) {
+                imageStopStream$.next();
+            }
 
```

Comment:
I think removing this event ` this.layer.getSource().on('imageloaderror', (event) => { ... });` could change the way errors are  handled for tiles.
`'imageloaderror'` seems related to tiles and not to layer.

Reply:
Mmm... I don't remember why I removed this (It passed something like a month). I'm going to restore the original one.